docs/DISTROS.md

```diff
@@ -0,0 +1,170 @@
+# curl distros
+
+<!-- markdown-link-check-disable -->
+
+Lots of organizations distribute curl packages to end users. This is a
+collection of pointers to where to learn more about curl on and with each
+distro.
+
+We discuss curl distro issues, patches and collaboration on the [curl-distros
+mailing list](https://lists.haxx.se/mailman/listinfo/curl-distros).
+
+# Alpine Linux
+
+- curl: https://pkgs.alpinelinux.org/package/edge/main/x86_64/curl
+- curl issues: https://gitlab.alpinelinux.org/alpine/aports/-/issues
+- curl security: https://security.alpinelinux.org/srcpkg/curl
+- curl package source and patches: https://gitlab.alpinelinux.org/alpine/aports/-/tree/master/main/curl
+
+## Arch Linux
+
+- curl: https://archlinux.org/packages/core/x86_64/curl/
+- curl issues: https://gitlab.archlinux.org/archlinux/packaging/packages/curl/-/issues
+- curl security: https://security.archlinux.org/package/curl
+- curl wiki: https://wiki.archlinux.org/title/CURL
+
```

Comment:
## Chimera

- curl package source and patches: https://github.com/chimera-linux/cports/tree/master/main/curl